Fixes for hugo 0.30 compatibility.

marekr opened the pull request:
Changes to work with hugo 0.30

Requires updating hugo on the server and locally to 0.30.

Nothing wrong the breadcrumbs :P Apparently we always did the menu weights wrong so that's fixed.

Per task item #220

nickoe commented on content/download/osx.adoc:
```diff
@@ -12,14 +12,14 @@ KiCad is supported on OS X 10.7 through OS X 10.12.
 - *KiCad extras* contains the footprints and all the instructions you need for making printed circuit board layouts while offline.
 
-=== Stable Release
+== Stable Release
```
Why is this header level changed? It looks like we use three for all the other pages.

marekr commented:
@nickoe I went and fixed all the warnings in the above commit. Changed the header levels. They look the same anyway and they make sense now, page title is <h1>, the adoc starts from <h2> instead of <h3>

nickoe commented:
Thank you!
